recipes/pythesint/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "pythesint" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "5e46853a8958bb3fbf26fba6eba0054551ac5004876740a941cf5e64faec2c4f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [not py27]
+
+requirements:
+  build:
+    - python
+    - pyyaml
+    - requests
+    - xdg  # [not win]
+    - setuptools
+  run:
+    - python
+    - pyyaml
+    - requests
+    - xdg  # [not win]
+
+test:
+  requires:
+    - mock
+    - nose
+  commands:
+    - nosetests pythesint
+
+about:
+  home: https://github.com/nansencenter/py-thesaurus-interface
+  license: GPL-3.0
+  license_family: GPL
+  ## license_file: LICENSE
```

Comment:
You need to include a license file for GPL packages. It's not currently packaged in the source, but you (as the package author) can fix that by adding `include LICENSE` to your `MANIFEST.in`. If you want to put it in conda-forge before making a new release including that, you can [package it manually](https://conda-forge.org/docs/meta.html#packaging-the-license-manually) until the next release comes out.

Reply:
Yes, this needs to be sorted out before we can merge

Reply:
Thanks for the replies. I will fix this.